package.json

```
@@ -29,8 +29,8 @@
   },
   "homepage": "https://github.com/pkgjs/wiby#readme",
   "dependencies": {
-    "@octokit/graphql": "^4.5.0",
-    "@octokit/rest": "^18.0.0",
+    "@octokit/graphql": "^8.0.0",
+    "@octokit/rest": "^21.0.0",
```

Since in version 21 it became an ESM-only package, it can't be used in this repository that easily yet.
```suggestion
    "@octokit/rest": "^20.0.0",
```